Client/TeamTalkClassic/TeamTalk.rc

```diff
@@ -336,6 +339,7 @@ CAPTION "TeamTalk"
 MENU IDR_MENU_MAIN
 FONT 8, "MS Shell Dlg", 0, 0, 0x0
 BEGIN
+    LTEXT           "Channel's list",IDC_STATIC_CHANLIST,66,79,11,8

```

Comment:
I had to remove this because it's overlapping the control.

Reply:
Oh, very bad news. There is no possibility to just move this in a better place ?
It's very problematic in this classic client for accessibility to find some areas without label :(

Reply:
Maybe we can make static text invisible. After that, I'll can test if screen reader interprete this invisible text correctly.